src/library/interface_stereo.cpp

```diff
@@ -33,11 +35,69 @@ void OrbSlam2InterfaceStereo::subscribeToTopics() {
       boost::bind(&OrbSlam2InterfaceStereo::stereoImageCallback, this, _1, _2));
 }
 
+void OrbSlam2InterfaceStereo::stereoRectification() {
+  // Load settings related to stereo calibration
+  cv::FileStorage fsSettings(settings_file_path_, cv::FileStorage::READ);
+
+  cv::Mat K_l, K_r, P_l, P_r, R_l, R_r, D_l, D_r, T_C0_1C, Q_;
+
+  fsSettings["LEFT.K"] >> K_l;
+  fsSettings["RIGHT.K"] >> K_r;
+
+  fsSettings["LEFT.P"] >> P_l;
+  fsSettings["RIGHT.P"] >> P_r;
+
+  fsSettings["LEFT.R"] >> R_l;
+  fsSettings["RIGHT.R"] >> R_r;
+
+  fsSettings["LEFT.D"] >> D_l;
+  fsSettings["RIGHT.D"] >> D_r;
+
+  fsSettings["T_c0_c1"] >> T_C0_1C;
+
+  int rows_l = fsSettings["LEFT.height"];
+  int cols_l = fsSettings["LEFT.width"];
+  int rows_r = fsSettings["RIGHT.height"];
+  int cols_r = fsSettings["RIGHT.width"];
+
+  if (K_l.empty() || K_r.empty() || P_l.empty() || P_r.empty() || R_l.empty() ||
+      R_r.empty() || D_l.empty() || D_r.empty() || rows_l == 0 || rows_r == 0 ||
+      cols_l == 0 || cols_r == 0) {
+    ROS_ERROR("Calibration parameters to rectify stereo are missing!");
+    return;
+  }
+
+  cv::stereoRectify(K_l, D_l, K_r, D_r, cv::Size(cols_l, rows_l),
+                    T_C0_1C.rowRange(0, 3).colRange(0, 3),
+                    T_C0_1C.col(3).rowRange(0, 3), R_l, R_r, P_l, P_r, Q_);
+
+  cv::initUndistortRectifyMap(K_l, D_l, R_l, P_l.rowRange(0, 3).colRange(0, 3),
+                              cv::Size(cols_l, rows_l), CV_32F, M1l_, M2l_);
+  cv::initUndistortRectifyMap(K_r, D_r, R_r, P_r.rowRange(0, 3).colRange(0, 3),
+                              cv::Size(cols_r, rows_r), CV_32F, M1r_, M2r_);
+
+  // Uncomment if you want to see the rectified instrinsics ex. to put in yaml
```

Comment:
In general code which you have to uncomment to use shouldn't be put into the master branch. Either expose the option to output these to the command line with a rosparam flag, or remove entirely. Of course this is often useful in development, so I encourage you to use it on a development branch.